guide/getting-started/software.md

```diff
@@ -36,66 +36,72 @@ Illustration sources
-   retina = "/assets/images/guide/getting-started/software/explorer-example@2x.jpg"
-   alt-text = "Illustrative interface for block explorer software"
-   caption = 'Block explorer offer insight into activiy on the bitcoin network. Texture by [Mike van den Bos](https://unsplash.com/@mike_van_den_bos){:target="_blank" rel="nofollow"} on [Unsplash](https://unsplash.com){:target="_blank" rel="nofollow"}.'
+   image = "/assets/images/guide/getting-started/software/wallet-example.jpg"
+   retina = "/assets/images/guide/getting-started/software/wallet-example@2x.jpg"
+   alt-text = "Illustrative interface of a wallet application"
+   caption = "Wallets allow users to send, receive, and manage transactions."
    width = 400
-   height = 388
+   height = 396
    layout = "float-right-desktop"
 %}
 
-If blockchain is a public database, block explorers are simply windows into that database. They let users view data about specific transactions, latest blocks, block height, and so on.
+Wallets are perhaps the most important Bitcoin applications. They provide easy-to-use graphical user interfaces (GUIs) designed for broad audiences that let bitcoin owners send, receive, store and manage their coins.
+
+Wallet features vary by application but always include wallet setup, balance and transaction records, and the ability to send and receive bitcoin. The full range of features that wallets may support is broad and includes security and privacy options, currency exchange features, accounting tools, interoperability, accessibility, and localization options.
 
-Explorers also provide insight into bigger picture activity on the Bitcoin network, such as daily transaction numbers. For example, there are typically fewer transactions on weekends, which results in lower fees, which are ideal for low-priority transactions.
+Few wallets support the full range of features. The reasons for this can vary from; standards not being available when the wallet was first developed (newer address formats or HD wallets for example), the choice to not include anything that implies trusting a third party, or simply because it does not fit the intended use case for the software.
+The features you include should be based on the needs of your users. Try to maximize interoperability with other Bitcoin products by supporting modern standards and emerging technologies. For example, a wallet project started today should almost certainly support the Lightning network.
+
+Due to Bitcoin’s open-source nature, anyone with the technical skills can develop a Bitcoin wallet. Many code libraries are available to simplify this task.
 
 </div>
 
-## Wallets
+## Exchanges
 
 <div class="center" markdown="1">
 
+Exchanges let users swap between currencies and networks (for example USD to bitcoin, or from the base layer to the Lightning network. They typically fall in three general categories. Consumer-focused exchanges let users make trades directly with the exchange. This is ideal for occasional use.
```

Comment:
```suggestion
Exchanges let users swap between currencies and networks (USD to bitcoin, or from the base layer to the Lightning network. They typically fall into three general categories. Consumer-focused exchanges let users make trades directly with the exchange. This is ideal for occasional use.
```

Reply:
This makes it read like these are the only swaps possible. I'll reject this for now as it also misses a ) that was added in another commit.